lib/PayPal/Rest/ApiContext.php

```diff
@@ -46,6 +52,60 @@ public function __construct($credential = null, $requestId = null)
     }
 
     /**
+     * Create new default ApiContext.
+     *
+     * @param \PayPal\Auth\OAuthTokenCredential $credential
+     * @param array $config
+     * @param string $payPalPartnerAttributionId
+     * @return ApiContext
+     */
+    public static function create($credential = null, $config=array(), $payPalPartnerAttributionId=null)
```

Comment:
I think we do not need this method. We can still go ahead and create an instance of ApiContext, and fill in the details.